src/bun.js/node/node_os.zig

```diff
@@ -314,4 +315,24 @@ pub const Os = struct {
         var name_buffer: [std.os.HOST_NAME_MAX]u8 = undefined;
         return JSC.ZigString.init(C.getVersion(&name_buffer)).withEncoding().toValueGC(globalThis);
     }
+
+    inline fn getMachineName() []const u8 {
+        return switch (@import("builtin").target.cpu.arch) {
+            .arm => "arm",
+            .aarch64 => "aarch64",
```

Comment:
Zig's name's don't align with node's names for these. For example `aarch64` is `arm64`
<img width="301" alt="image" src="https://user-images.githubusercontent.com/709451/200480570-3dcca3f9-34ca-4f43-a432-c337dd0068da.png">


We need it to align with node's names for these so that libraries those values work as expected

Reply:
Really interesting.  The documentation seems to incorrect as it calls it `aarch64` and fails to list `arm64` as a possible option.